.github/workflows/metrics-deploy.yml

```diff
@@ -0,0 +1,143 @@
+name: Aztec Metrics Stack Deployment
+
+on:
+  workflow_call:
+    inputs:
+      namespace:
+        description: The namespace to deploy to, e.g. metrics
+        required: true
+        type: string
+        default: metrics
+      values_file:
+        description: The values file to use, e.g. 1-validators.yaml
```

Comment:
It's a nit but presumably this example should be one of the metrics values files?